apsuite/commisslib/topup_feedbacks.py

```diff
@@ -0,0 +1,315 @@
+"""."""
+import time as _time
+
+from epics.ca import CAThread as _Thread
+import numpy as np
+import GPy as gpy
+
+from mathphys.functions import get_namedtuple
+from siriuspy.devices import HLTiming, EGun, InjCtrl, CurrInfoAS
+
+from ..utils import ThreadedMeasBaseClass as _BaseClass, \
+    ParamsBaseClass as _ParamsBaseClass
+
+_DTMP = '{0:26s} = {1:9d}  {2:s}\n'.format
+_STMP = '{0:26s} = {1:9s}  {2:s}\n'.format
+_FTMP = '{0:26s} = {1:9.2f}  {2:s}\n'.format
+np_poly = np.polynomial.polynomial
```

Comment:
is your intention to expose this symbol in the module?

Reply:
No it is not. I just don't pay too much attention to what is exposed in apsuite just as I pay in siriuspy. I'll change it, thanks!

Reply:
done!